example/src/screens/calendars.tsx

```diff
@@ -24,6 +24,7 @@ const CalendarsScreen = () => {
       <Fragment>
         <Text style={styles.text}>Calendar with selectable date</Text>
         <Calendar
+          disableAllTouchEventsForInactiveDays
```

Comment:
Can you please have a separate example for the inactive state?

Reply:
sure